device/bgp/evpn/eviroutedistinguisher.yaml

```diff
@@ -0,0 +1,25 @@
+components:
+  schemas:
+    Bgp.EviRouteDistinguisher:
```

Comment:
should not we make it generic for bgp like RouteDistinguisher? and then whoever needs it just refer it. file should be renamed and placed outside evi (under bgp).